Meetups/27/2025-06-05-minutes.md

```diff
@@ -0,0 +1,131 @@
+
+## Info
+
+:date: **Date:** 27 March 2025
+
+### :bust_in_silhouette: Participants
+
+<!-- This list will copied over from the meeting tool -->
+- Ege Korkan
+- Cristiano Aguzzi
+- Thomas Wehr
+- Jan Hofmann
+- Kunihiko Toumura
+- Thomas Jaeckle
+- Zhou Gui
+- Andreas Harth
+- Jonathan Wise
+- Tatiana Mangels
+- York Schmidtke
+- Daniel Peintner
+
+:writing_hand: **Scribe:** Cris
+
+----
+
+### Introduction
+
+Ege: Welcome Everyone! we are at Meetup 27th! All slide are public
+Ege: you can find us in different places on the web
+
+### News
+
+Ege: we do not have new meetups planned in the future but you can join the office hours to talk with us.
+
+Ege: we have two new tutorial videos! please visit our youtube channel
+
+Ege: there are more underway, if you are interested we are meeting every friday. You can help us with new ideas for tutorials
+
+Ege: This year TPAC is planned in Kyoto, Japan, for 10-14 November. We are going to join as CG please let us know if you want to attend.
+
+Ege: join the CG to keep in touch or to propose a new meetup!
+
+### Meetup Presentation
+
+Ege: Today we have Thomas Wehr, he is a PhD at the Chair for Technical Information Systems. In the past his department have participated in different WoT projects.
+
+Thomas: We developed a tool to visual create WoT consumer. We wanted to create a programming environment that should portable and usable as many devices as possible. Moreover, it should be very accessible (that's why we opted for a visual programming interface). We use the Web of Things to make interoperable with many type of devices.
+
+Thomas: visual programming in general refers to a programming environment that you arrange graphical elements to create programs (workflow, flowchart, and block based). We opted for a block based approach and we called Blast.
+
+Thomas: we use Blockly from Google and we adapted it to Web of Things concepts
+
+Thomas: there are many pre-defined blocks in Blockly: string operations, mathematical operations, speech to text and vice versa, http requests, etc.
+
+Thomas: Blast has the concept of state. The main feature is to consume a Thing Description and change the device without changing any code. Blast can list only the properties available in the TD and also can validate the input/output of a block using TD data schema.
+
+Thomas: Blast is built on top of node-wot. It worked out-of-the-box for us, but we needed a protocol binding for Bluetooth devices. The second extension that we had to do is to support "scale" and "default". The "scale" was useful for a particular device called RuuviTag and the default for a BLE LED controller. The default keyword was useful to write concise data payloads.
+
+Thomas: let's see Blast in the action.
+
+> Shows Blast demo
+> Thomas program a Stream deck to connect with a Lamp
+
+Thomas: to concluded that was Blast a visual programming and execution environment for WoT Consumers. It can consume TDs to dynamically to add new devices.
```

Comment:
```suggestion
Thomas: To conclude, that was Blast, a visual programming and execution environment for WoT Consumers. It can consume TDs to dynamically to add new devices.
```